lib/Runtime/Language/SIMDInt8x16OperationX86X64.cpp

```diff
@@ -149,8 +149,18 @@ namespace Js
         X86SIMDValue x86Result;
         X86SIMDValue tmpaValue = X86SIMDValue::ToX86SIMDValue(aValue);
         X86SIMDValue tmpbValue = X86SIMDValue::ToX86SIMDValue(bValue);
-        x86Result.m128i_value = _mm_max_epi8(tmpaValue.m128i_value, tmpbValue.m128i_value);  //  max(a,b) == a
-        x86Result.m128i_value = _mm_cmpeq_epi8(tmpbValue.m128i_value, x86Result.m128i_value); //
+        if (AutoSystemInfo::Data.SSE4_1Available())
+        {
+            x86Result.m128i_value = _mm_max_epi8(tmpaValue.m128i_value, tmpbValue.m128i_value);  //  max(a,b) == a
+            x86Result.m128i_value = _mm_cmpeq_epi8(tmpbValue.m128i_value, x86Result.m128i_value); //
+        }
+        else
+        {
+            X86SIMDValue x86Tmp1, x86Tmp2;
+            x86Tmp1.m128i_value = _mm_cmplt_epi8(tmpaValue.m128i_value, tmpbValue.m128i_value); // compare a > b?
```

Comment:
Another solution is to always include and execute the JS polyfill with ChakraCore, that way we have the functionality on non-SSE2 devices. Would that be possible ?